gr-digital/python/digital/qa_ofdm_chanest_vcvc.py

```diff
@@ -301,10 +301,11 @@ def test_999_all_at_once(self):
 
         def run_flow_graph(sync_sym1, sync_sym2, data_sym):
             top_block = gr.top_block()
-            carr_offset = random.randint(-max_offset / 2, max_offset / 2) * 2
+            carr_offset = random.randint(-max_offset // 2, max_offset // 2) * 2
             tx_data = shift_tuple(sync_sym1, carr_offset) + \
                 shift_tuple(sync_sym2, carr_offset) + \
-                shift_tuple(data_sym, carr_offset)
+                shift_tuple(data_sym, carr_offset) + \
+                tuple((0,) * fft_len)
```

Comment:
I'm a bit surprised by this. The first input vector to ofdm_chanest_vcvc should be the first sync symbol, in frequency domain. If it's all zero, then this should fail. Are we still losing data?

Reply:
That seems to be the case, on some runs, sadly, for me. Haven't rebuilt this branch lately, though.